packages/apps/shopify-api/lib/auth/oauth/__tests__/migrate-to-expiring-tokens.test.ts

```diff
@@ -0,0 +1,124 @@
+import * as ShopifyErrors from '../../../error';
+import {DataType, shopifyApi} from '../../..';
+import {testConfig} from '../../../__tests__/test-config';
+import {queueMockResponse} from '../../../__tests__/test-helper';
+import {RequestedTokenType} from '../token-exchange';
+
+let shop: string;
+let nonExpiringOfflineAccessToken: string;
+
+beforeEach(() => {
+  shop = 'someshop.myshopify.io';
+  nonExpiringOfflineAccessToken = 'some-access-token';
+});
+
+describe('migrateToExpiringToken', () => {
+  describe('with valid parameters', () => {
```

Comment:
Do we need t consider what happens when:

1. Params are invalid?
2. The user attempts to migrate a token that has already been migrated?